src/main/java/com/spotinst/sdkjava/model/SpotOceanEcsClusterClient.java

```diff
@@ -0,0 +1,141 @@
+package com.spotinst.sdkjava.model;
+
+import com.spotinst.sdkjava.exception.HttpError;
+import com.spotinst.sdkjava.exception.SpotinstHttpException;
+import com.spotinst.sdkjava.model.bl.ocean.ecs.ClusterLaunchSpecification;
+import com.spotinst.sdkjava.model.bl.ocean.ecs.OceanEcsClusterLaunchSpecRequest;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+public class SpotOceanEcsClusterClient {
```

Comment:
Should this be named as SpotOceanEcsLaunchSpecClient?

Reply:
This client is common for launch spec and cluster. Hence not updating the class name